src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1252,18 +1269,20 @@ public Worker build() {
                 }
             }
             if (metricsFactory == null) {
-                metricsFactory = getMetricsFactory(cloudWatchClient, config);
+                metricsFactory = Worker.getMetricsFactory(cloudWatchClient, config);
```

Comment:
Why is this being changed? The getMetricsFactory method is a static method which is accessible within the same class.

Reply:
It was changed because the Lombok Getter had the same name as the static method.